lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1339,6 +1339,13 @@ enum AddressMaskRange {
   eAddressMaskRangeAll = eAddressMaskRangeAny,
 };
 
+/// Useful for callbacks whose return type indicates
+/// whether to continue iteration or short-circuit.
+enum class IterationAction {
+  eContinue = 0,
+  eStop,
+};
+
```

Comment:
We don't need this in the public API, so this should be moved to `lldb-private-enumerations.h`

since this is a `enum class` do we need the leading `e` character? So it would just be:
```
enum class IterationAction {
  Continue = 0,
  Stop,
};
```

Reply:
done